scripts/lib/node_cache.py

```diff
@@ -50,21 +55,28 @@ def setup_node_modules(npm_args=None, stdout=None, stderr=None, copy_modules=Fal
     for cmd in cmds:
         run(cmd, stdout=stdout, stderr=stderr)
 
-def do_npm_install(target_path, npm_args, success_stamp, stdout=None, stderr=None,
-                   copy_modules=False):
-    # type: (str, List[str], str, Optional[IO], Optional[IO], Optional[bool]) -> None
+def do_yarn_install(target_path, yarn_args, success_stamp, stdout=None, stderr=None,
+                    copy_modules=False, prefer_offline=False):
+    # type: (str, List[str], str, Optional[IO], Optional[IO], Optional[bool], Optional[bool]) -> None
     cmds = [
-        ["rm", "-rf", target_path],
```

Comment:
Rerunning should work. It was specifically designed to be idempotent. I am cool with assuming that we don't need to remove the directories unless proven otherwise.